### PR TITLE
[WIP] Added Longest Common Sequence  

### LINKED ISSUE #308: Longest common subsequence
Description of the problem
LCS Problem Statement: Given two sequences, find the length of longest subsequence present in both of them. A subsequence is a sequence that appears in the same relative order, but not necessarily contiguous.
Example of the problem
For example, “abc”, “abg”, “bdf”, “aeg”, ‘”acefg”, .. etc are subsequences of “abcdefg”.

### PR DESCRIPTION
**Pull request by LazarusCoder**
#### References to other Issues or PRs or Relevant literature

Fixes  #308 #310 

#### Brief description of what is fixed or changed
Longest Common Subsequence #308  is been added with backtracking 
so now we  get the Lenght of LCS and sequence too

### CODE REVIEW

**Comment by czgdp1807** [comment]
Please go through some already present code in `master`. The code is okay but the function names aren't descriptive, the document string isn't following the already existing style. Code quality checks are failing.

**Comment by LazarusCoder** [comment]
LCS with Back Tracking Done

**Comment by sidhu1012** [comment]
> Please go through some already present code in `master`. The code is okay but the function names aren't descriptive, the document string isn't following the already existing style. 

These issues still persist in code. Please make function name more informative and follow numpy docstring guidelines.

**Comment by LazarusCoder** [comment]
> > Please go through some already present code in `master`. The code is okay but the function names aren't descriptive, the document string isn't following the already existing style.
> 
> These issues still persist in code. Please make function name more informative and follow numpy docstring guidelines.

right now function name is LCS =>Longest common sub sequence would be big as a function name same while calling it dont you think?

File: pydatastructs/linear_data_structures/algorithms.py [hunk 1]
```diff
@@ -546,3 +547,54 @@ def bucket_sort(array: Array, **kwargs) -> Array:
     if _check_type(array, DynamicArray):
         array._modify(force=True)
     return array
+
+def LCS(data_A,data_B):
```
Comment (czgdp1807) [inline pydatastructs/linear_data_structures/algorithms.py:551]:
```suggestion
def lowest_common_subsequence(seq1, seq2):
```

**Comment by czgdp1807** [comment]
Care to add some spaces around `=`, after, `,`. It just makes it easier to read code.

**Comment by sidhu1012** [comment]
It has conflicts now. Please resolve them.

**Comment by czgdp1807** [comment]
Closing in favour of https://github.com/codezonediitj/pydatastructs/pull/315

**Pull request closed without merging**